vite.config.ts

```diff
@@ -35,7 +35,11 @@ export default defineConfig({
     },
     test: {
         coverage: {
-            exclude: ["commitlint.config.ts", "docs"],
+            exclude: [
+                ...coverageConfigDefaults.exclude,
```

Comment:
We were getting coverage reports on our tests 😁

Reply:
woops. good catch, thanks!